src/node_worker.cc

```diff
@@ -98,8 +138,7 @@ Worker::Worker(Environment* env,
 }
 
 bool Worker::is_stopped() const {
-  Mutex::ScopedLock stopped_lock(stopped_mutex_);
-  return stopped_;
+  return thread_stopper_->IsStopped();
```

Comment:
Suggestion: Could you add a `CHECK(thread_stopper_);`?

Reply:
done!